README.org

```diff
@@ -71,8 +71,11 @@
 ** Packages, Crates and modules (Quest 5 -> 1 day)
    Learning how to organize projects separating them into components (packages and modules), unit testing and importing crates
    - [[file:drawing/src/main.rs][modules]]
-   - [[file:drawing/src/main.rs][drawing]]
-     Total: 2
+   - [[file:drawing/src/main.rs][does_it_fit]]
+   - [[file:drawing/src/main.rs][macro_calculator]]
+   - [[file:drawing/src/main.rs][expected_variable]]
+   - [[file:drawing/src/main.rs][mobs]]
```

Comment:
I think you forgot to update the name of the file in the link. All those exercises are not in drawing/src/main.rs